.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+conda:
+   environment: docs/environment.yaml
+
+sphinx:
+   configuration: docs/conf.py
+
+submodules:
+   include: all 
+
+build:
+   os: "ubuntu-lts-latest" # https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os
```

Comment:
This is a new feature, but will ensure that the build always uses the most up-to-date platforms provided by RTD.